metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/usage.py

```diff
@@ -67,6 +67,7 @@
     "DELETE": OperationTypeClass.DELETE,
     "MERGE": OperationTypeClass.UPDATE,
     "CREATE": OperationTypeClass.CREATE,
```

Comment:
Seems like we can remove this: I don't see it in either of the links above, oops!

Reply:
yes, will do.